doc.go

```diff
@@ -106,7 +106,7 @@ The following connection parameters are supported:
 
   - application: Identifies your application to Snowflake Support.
 
-  - insecureMode: false by default. Set to true to bypass the Online
+  - disableOCSPChecks: false by default. Set to true to bypass the Online
```

Comment:
we should keep both in docs, just `insecureMode` should be marked as deprecated

Reply:
Ok, done.